quickref/qsharp-quick-reference.tex

```diff
@@ -342,6 +609,73 @@
     Run all unit tests                & \texttt{dotnet test} \\
   \end{keysref}
 
+	\columnbreak
+	\section{Math reference}
+
+	\begin{keysref}{Complex Arithmetic}
+    	\(i^2 \) & \(-1\) \\
+	\( (a + bi) + (c + di) \) & \( (a + c ) + (b + d)i \) \\
+	\( (a + bi)  (c + di) \) & \( a \cdot b + a \cdot di + c \cdot bi + (b \cdot d)i^2 \) \newline
+		\(  a \cdot b + a \cdot di + c \cdot bi - (b \cdot d) \) \\
+
+	Complex Conjugate \newline \(x = a + bi \) & \( \overline{x} = a - bi \) \\
+ 	Complex division \newline \( \frac{a +bi}{c+di} \) & \( \frac{x}{y} =  \frac{x}{y} \cdot 1 = \frac{x}{y} \cdot \frac{\overline{y}}{\overline{y}} \) \newline
+		\( = \frac{(a+bi)(c-di) }{c+di)(c-di)} = \frac{(a+bi)(c+di)}{c^2 + d^2} \) \\
+	Modulus \newline \( x = a + bi \) & \( |x| = \sqrt{a^2 + b^2} \) \\
+	\(e^{i\theta} \) & \( \cos(\theta)  + i \sin(\theta) \) \\ 
+ 	\( e^{a + bi} \) & \( e^a + e^{bi} = e^a(\cos(b) + i\sin(b)) \)  \\
+	\( r^{a+bi} \) & \( c^a \cdot c^{bi} = c^a \cdot e^{bi\ln(c)} \) \newline \(= c^a((\cos(b\ln(c)) + i\sin(b\ln(c))) \)\\
+	Polar form \newline \(a+bi \) & \( re^{i\theta} = r(\cos(\theta) + i\sin(\theta))  \) \newline
+		\( r =  \sqrt{a^2 + b^2} \) \newline \( \theta = \tan^{-1}(\frac{b}{a}) \) \\
+	\end{keysref}
+
+	\begin{keysref}{Linear Algebra}
+	\usebox\matrixadditionleft & \usebox\matrixadditionright \\
+	
+	\usebox\matrixscalarleft &\usebox\matrixscalarright \\
+	
+	\usebox\matrixmultiplicationleft & \usebox\matrixmultiplicationright \\
+
+	\usebox\matrixtransposeleft & \usebox\matrixtransposeright \\	 
+	
+	\usebox\matrixadjointleft & \usebox\matrixadjointright \\
+
+	\usebox\innerproductleft & \usebox\innerproductright \\ 	
+	
+	\usebox\outerproductleft & \usebox\outerproductright \\
+	
+\end{keysref}
+
+\newpage
+
+\section{Gates reference}
+
+
+\begin{keysrefQbits}{Single Qubit gates}
+
+	Gate & Matrix representation    & Ket-Bra & Applying to \( |\psi\rangle = \alpha|0\rangle + \beta|1\rangle \) & Applying to basis states & & \\
+	
+	 X & \usebox\xgate & \( |0\rangle\langle1| + |1\rangle\langle0|\) & \( |\psi\rangle = \alpha|1\rangle + \beta|0\rangle \) & \( X|0\rangle = |1\rangle \newline X|1\rangle = |0\rangle \) & \(
```

Comment:
I would do a couple of purely cosmetic changes to this section to make the life easier for the next person who edits it:
* add an empty line before each new gate (they don't show up in the table and make navigation a lot easier)
* split each line into several shorter lines (column breaks are good places to break the line)
* use \ket{...} and \bra{...} macros instead of |...\rangle and \langle...| - not all environments support them, but this one does.